introduction.md

```diff
@@ -0,0 +1 @@
+Hi I'm Arch and I am a software engineering student at Bloc.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.